gatsby/package.json

```diff
@@ -33,7 +33,7 @@
   "dependencies": {
     "@nice-digital/browserslist-config": "^2.0.1",
     "@nice-digital/design-system": "^1.4.0",
-    "@nice-digital/global-nav": "^4.1.711",
+    "@nice-digital/global-nav": "^4.1.712-GN-167-reduce-autoco",
```

Comment:
Can you bump to main build now that https://github.com/nice-digital/global-nav/pull/167 is merged please?